src/components/credentials-popover/credentials-popover.component.jsx

```diff
@@ -0,0 +1,62 @@
+/*
+ * Copyright: Copyright 2023 SAP SE or an SAP affiliate company and cdc-tools-chrome-extension contributors
+ * License: Apache-2.0
+ */
+
+import { useSelector, useDispatch } from 'react-redux'
```

Comment:
the folder src/components/credentials-popover was removed in the main branch, it should not be added again